web/client/utils/LocaleUtils.js

```diff
@@ -51,6 +51,18 @@ const DATE_FORMATS = {
     "it-IT": "dd/MM/yyyy",
     "nl-NL": "dd/MM/yyyy"
 };
+
+const servicesErrorParser = {
+    geostore: require('../api/GeoStoreDAO').errorParser
```

Comment:
Can we do the opposite?
 * registerParser = (type, parser) => {} in LocaleUtils
 * registerParser('geostore', parser) in GeoStoreDAO